src/test/clojure/clara/test_accumulators.clj

```diff
@@ -59,10 +59,15 @@
         session (-> empty-session
                     (insert (->Temperature 30 "MCI"))
                     (insert (->Temperature 10 "MCI"))
-                    (insert (->Temperature 80 "MCI")))
+                    (insert (->Temperature 80 "MCI"))
+                    fire-rules)
```

Comment:
a lot of tests to update to add a fire-rules call
do we have concerns around how this would impact consumers necessitating a call to fire rules?